swatch-system-conduit/schemas/inventory/hbi-system-profile.yaml

```diff
@@ -37,4 +37,5 @@ properties:
       $ref: "hbi-network-interface.yaml"
   virtual_host_uuid:
     type: string
-
+  conversions_activity:
+    type: string
```

Comment:
Per: https://github.com/RedHatInsights/inventory-schemas/pull/129/files#diff-952f7dbba2e7a294392a7ac96f3f9a505fff472c194ecf63763c221e7ef84fb5R759, I believe the value in the HBI system profile is a boolean